documentation/cml_on_hyperv.md

```diff
@@ -0,0 +1,58 @@
+# CML on Hyper-V
+This installs CML on Hyper-V for Windows. You will need a version of windows that supports and provides Hyper-V as a feature. Editions other than Pro and Enterprise likely do not include this feature. Server 2016 and later should include Hyper-V, although the process below may change slightly if you are running this on a server instance.
+
+This was tested on Windows 11 Enterprise, but should work for other editions of windows that meet the requirements.
+
+## Requirements
+- Windows 10 Pro/Enterprise or later, Server 2016 or later
+- Hyper-V windows feature activated
+
+## Setup
+### Enable Hyper-V
+- Windows Settings: "Turn Windows features on or off"
+- Enable all Hyper-V options
+- Reboot when asked
+
+### Create Hyper-V Bridged Network
+This is done to expose the VM to your local network. If you only plan on accessing the CML instances from your local machine only, you can skip this step.
+- Launch the Hyper-V GUI
+- Click the "Virtual Switch Manager"
+- Create a new "External" virtual switch
+- Pick a network card connected to your local network
+- Confirm and save
+
+### Create Hyper-V Virtual Machine
+Before proceeding with the following steps, make sure to download the latest CML ISO for bare metal installation.
```

Comment:
You can reference https://developer.cisco.com/docs/modeling-labs/downloading-files-for-cml-installation/ here.  You may also need to indicate that it is a _different_ ISO file from the refplat ISO.  (You'd be surprised how many people make that mistake.)